languages/python/oso/polar/polar.py

```diff
@@ -144,6 +144,18 @@ def load_file(self, policy_file):
 
     def load_str(self, string, filename=None):
         """Load a Polar string, checking that all inline queries succeed."""
+        # Get MRO of all registered classes
+        # NOTE: not ideal that the MRO gets updated each time load_str is
+        # called, but since we are planning to move to only calling load once
+        # with the include feature, I think it's okay for now.
+        for (cls_name, cls) in self.host.classes.items():
+            mro = [
+                self.host.class_ids.get(c)
+                for c in inspect.getmro(cls)
+                if self.host.class_ids.get(c)
```

Comment:
```suggestion
                if c in self.host.class_ids
```

Would be the typical way I'd write this